RecipeApi.Tests/DatabaseTest/RecipeDbTest.cs

```diff
@@ -24,15 +24,13 @@ public void DbContext_CanAddAndRetrieveRecipes()
                 var recipe = new Recipe { Title = "Test Recipe", Portions = 4, TimeToCook = "30 min", CreatedAt=DateTime.Now, Description = "Test Description"};
                 context.Recipes.Add(recipe);
                 context.SaveChanges();
-            }
 
-            // Assert
-            using (var context = new RecipeDbContext(options))
-            {
                 var retrievedRecipe = context.Recipes.FirstOrDefault(r => r.Title == "Test Recipe");
                 Assert.NotNull(retrievedRecipe);
                 Assert.Equal("Test Description", retrievedRecipe.Description);
                 Assert.Equal("Test Recipe", retrievedRecipe.Title);
+
+                context.Recipes.Remove(retrievedRecipe);
```

Comment:
It's better to use xUnit attributes to clean db before each test or set of tests.
You can create a base class and then inherit all tests which needs to clean db before execution.
https://hamidmosalla.com/2018/08/30/xunit-beforeaftertestattribute-how-to-run-code-before-and-after-test/